newIDE/app/src/ObjectsList/index.js

```diff
@@ -274,6 +470,25 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
     }: Props,
     ref
   ) => {
+    // TODO Handle any number of object containers.
+    const objectsContainersList = projectScopedContainersAccessor
+      .get()
+      .getObjectsContainersList();
+
+    if (objectsContainersList.getObjectsContainersCount() === 0) {
+      throw new Error('Used ObjectsList without any object container.');
+    }
+    if (objectsContainersList.getObjectsContainersCount() > 2) {
+      console.error('Used ObjectsList with more than 2 object containers.');
+    }
+    const globalObjectsContainer =
+      objectsContainersList.getObjectsContainersCount() > 1
+        ? objectsContainersList.getObjectsContainer(0)
+        : null;
+    const objectsContainer = objectsContainersList.getObjectsContainer(
+      objectsContainersList.getObjectsContainersCount() - 1
+    );
```

Comment:
Should this logic of splitting the global objects container and "the second one" as the "local" objects container be in a dedicated method?

Reply:
It could but it's not a lot of code and it will be removed when when the component handles N object containers.